[miopen-opencl-5.1.1] linker error

miopen-opencl-5.1.1 does not build for me in a clean chroot:
```
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirGetKernelCount'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirGetExecutionDims'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirDestroyHandle'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirLowerTuningParams'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirBufferGet'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirLowerBin'
/usr/bin/ld: ../lib/libMIOpen.so.1.0: undefined reference to `miirCreateHandle'
clang-14: error: linker command failed with exit code 1 (use -v to see invocation)
```

Dependencies that were installed:
```
Packages (41) comgr-5.1.1-1  hip-runtime-amd-5.1.1-1  hsa-amd-aqlprofile-bin-5.1.1-1  hsa-rocr-5.1.1-1  hsakmt-roct-5.1.1-1  hwdata-0.358-1  kmod-29-3
              libdrm-2.4.110-1  libedit-20210910_3.1-1  libglvnd-1.4.0-1  libnsl-2.0.0-2  libomxil-bellagio-0.9.3-3  libpciaccess-0.16-2  libunwind-1.6.2-1
              libx11-1.7.5-1  libxau-1.0.9-3  libxcb-1.14-1  libxdamage-1.1.5-3  libxdmcp-1.1.3-3  libxext-1.3.4-3  libxfixes-6.0.0-1  libxshmfence-1.3-2
              libxxf86vm-1.1.4-4  llvm-libs-13.0.1-2  lm_sensors-1:3.6.0.r41.g31d1f125-1  mesa-22.0.1-3  numactl-2.0.14-2  openmp-13.0.1-1  pciutils-3.7.0-2
              python-3.10.4-1  rocm-device-libs-5.1.1-1  rocminfo-5.1.1-1  vulkan-icd-loader-1.3.208-1  wayland-1.20.0-2  xcb-proto-1.14.1-5
              xorgproto-2021.5-1  z3-4.8.14-1  ocl-icd-2.3.1-1  rocblas-5.1.1-1  rocm-llvm-5.1.1-1  rocm-llvm-mlir-5.1.0-1
```
makedepends:
```
Packages (10) boost-libs-1.78.0-1  hicolor-icon-theme-0.17-2  jsoncpp-1.9.5-1  libuv-1.43.0-1  rhash-1.4.2-1  boost-1.78.0-1  cmake-3.23.0-1
              miopengemm-5.1.1-1  opencl-headers-2:2022.01.04-1  rocm-cmake-5.1.1-1
```

--- a/rocm-llvm-mlir/PKGBUILD
+++ b/rocm-llvm-mlir/PKGBUILD
@@ -6,7 +6,7 @@ pkgname=rocm-llvm-mlir
 pkgdesc="Radeon Open Compute - LLVM Multi-Level IR Compiler Framework"
 _pkgver=5.1
 pkgver=5.1.0
-pkgrel=1
+pkgrel=2
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/llvm-project-mlir"
 license=('custom:Apache 2.0 with LLVM Exception')
@@ -14,6 +14,7 @@ depends=("hip")
 makedepends=("cmake" "sqlite" "python")
 source=("$pkgname-$pkgver.tar.gz::$url/archive/release/rocm-$_pkgver.tar.gz")
 sha256sums=('7bc1eba4af74a4884efcf1cc73933efd7d8bc97445bf2a2e441d4db4ed624c4f')
+options=(!lto)
 _dirname="$(basename "$url")-release-$(basename "${source[0]}" .tar.gz)"
 
 build() {

